numpy/random/randomgen/generator.pyx

```diff
@@ -1138,7 +1002,7 @@ cdef class RandomGenerator:
         """
         random_integers(low, high=None, size=None)
 
-        Random integers of type np.int64 between `low` and `high`, inclusive.
+        Random integers of type np.int between `low` and `high`, inclusive.
```

Comment:
On 32 bit systems this gives `int32`?

Reply:
Yes, also on all bitness Windows.  This function is deprecated and IMO should be removed before this transition.